content/guides/production-build.md

```diff
@@ -71,12 +74,14 @@ module.exports = {
   /*...*/
 };
 ```
-##Configuring webpack for multiple environments
+## Configuring webpack for multiple environments
 
 When we do have multiple configurations in mind for different environments, the easiest way is to write seperate js files for 
 each environment. For example:
-####config/dev.js
+
 ```js
+//config/dev.js
```

Comment:
You could handle the filenames likethis:

**config/dev.js**

```js
...
```

So bold on a separate line.

Apart from that this is good.